README.md

```diff
@@ -139,7 +139,12 @@ For guidance on installation, development, deployment, and administration, see o
 
 <h2><img height="20" src="/img/installation.svg">&nbsp;&nbsp;Installation</h2>
 
-SurrealDB is designed to be simple to install and simple to run - using just one command from your terminal. In addition to traditional installation, SurrealDB can be installed and run with HomeBrew, Docker, or using any other container orchestration tool such as Docker Compose, Docker Swarm, Rancher, or in Kubernetes.
+Sure, here is the paragraph converted into a point-wise format:
```

Comment:
> Sure, here is the paragraph converted into a point-wise format:

I get the impression you used an AI chatbot to revise the wording and forgot to remove this part of the response :laughing:

Reply:
Yes 😅 i forgot it, AI helps a lot actually 😅